lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/gateway/KBVGateway.java

```diff
@@ -55,9 +59,14 @@ public class KBVGateway {
 
     @Tracing
     public QuestionsResponse getQuestions(QuestionRequest questionRequest) {
-        TracingUtils.putAnnotation(EXPERIAN_IIQ_REQUEST, "saa");
+        TracingUtils.putAnnotation(EXPERIAN_IIQ_REQUEST, EXPERIAN_START_AUTHENTICATION_ATTEMPT);
```

Comment:
If this putAnnotation method isn't serving any purpose at the moment then it can be removed